docs/DeveloperGuide.md

```diff
@@ -450,36 +450,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User requests to list persons.
-2.  Arrow shows a list of persons.
-3.  User requests to list tasks of a specific person in the persons list.
-4.  Arrow shows a list of task of the person.
-5.  User requests to mark a specific task in the tasks list as complete or incomplete.
-6.  Arrow marks the task as specified.
+1.  User requests to mark a task to be complete.
+2.  Arrow shows the task to be complete.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The persons list is empty.
-
-  Use case ends.
-
-* 3a. The given persons index is invalid.
+* 1a. The given task index is invalid.
 
-    * 3a1. Arrow shows an error message.
+    * 1a1. Arrow shows an error message.
 
       Use case resumes at step 2.
       
-* 4a. The tasks list is empty.
+* 2a. The tasks list is empty.
```

Comment:
I think we could either choose to remove this extension
or add new steps in our MSS to 1. User requests to list tasks 2. Arrow shows a list of tasks